public/app/features/explore/Table/TableContainer.tsx

```diff
@@ -93,27 +93,29 @@ export class TableContainer extends PureComponent<Props> {
             {() => <MetaInfoText metaItems={[{ value: t('explore.table.no-data', '0 series returned') }]} />}
           </PanelChrome>
         )}
-        {frames &&
-          frames.length > 0 &&
-          frames.map((data, i) => (
-            <PanelChrome
-              key={data.refId || `table-${i}`}
-              title={this.getTableTitle(dataFrames, data, i)}
-              width={width}
-              height={this.getTableHeight(data.length, this.hasSubFrames(data))}
-              loadingState={loading ? LoadingState.Loading : undefined}
-            >
-              {(innerWidth, innerHeight) => (
-                <Table
-                  ariaLabel={ariaLabel}
-                  data={data}
-                  width={innerWidth}
-                  height={innerHeight}
-                  onCellFilterAdded={onCellFilterAdded}
-                />
-              )}
-            </PanelChrome>
-          ))}
+        {frames && frames.length > 0 && (
+          <div style={{ display: 'flex', flexDirection: 'column', gap: '8px' }}>
```

Comment:
this is grea! 

can we use the theme for this?
you can do that by using th `withTheme2` HOC:

```
export default withTheme2(connector(YourComponent));
```

then you'll have access to a `theme` prop, so here you can do something like 


```
<div className={css({ display: 'flex', flexDirection: 'column', gap: theme.spacing(1) })}>
```

you can have a look at `public/app/features/explore/Logs/LiveLogs.tsx` for a. more concrete example, or search for `withTheme2(` in the codebase, you should get quite a few results :)

lmk if that doesn't work

Reply:
@Elfo404, Thanks for the feedback. I have used `withTheme2` and also updated the test.